src/main/java/org/example/Twitter.java

```diff
@@ -0,0 +1,461 @@
+    ArrayList<Integer> followers = new ArrayList<Integer>();
+    ArrayList<Integer> following = new ArrayList<Integer>();
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public void setUserId(int userId) {
+        this.userId = userId;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getHandle() {
+        return handle;
+    }
+
+    public void setHandle(String handle) {
+        this.handle = handle;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public ArrayList<Integer> getTweets() {
+        return tweets;
+    }
+
+    public void setTweets(ArrayList<Integer> tweets) {
+        this.tweets = tweets;
+    }
+
+    public ArrayList<Integer> getFollowers() {
+        return followers;
+    }
+
+    public void setFollowers(ArrayList<Integer> followers) {
+        this.followers = followers;
+    }
+
+    public ArrayList<Integer> getFollowing() {
+        return following;
+    }
+
+    public void setFollowing(ArrayList<Integer> following) {
+        this.following = following;
+    }
+
+} 
+
+
+public class Twitter
+{
+    public static int currentUserId = -1;
+    public static int tweetCounter = 0;
+    public static LinkedHashMap<String, Tweet> tweetDict = new LinkedHashMap<String, Tweet>();
+    public static LinkedHashMap<String, User> userDict = new LinkedHashMap<String, User>();
+
+
+    public static void main(String args[])
+    {
+        Scanner myObj = new Scanner(System.in);
+        System.out.println("Hello, Twitter");
+        try {
+            printMenu();
+            parseJsonInfo("database.json");
+
+            // If the login is correct, continue with Twitter
+            String userName = myObj.nextLine();
+            String passWord = myObj.nextLine();
+
+            if(login(userName, passWord) != -1){
+                start();
+            }
+        } catch (IOException | ParseException fnfe) {
+            fnfe.printStackTrace();
+        }
+    }
+
+    public static void printMenu() throws java.io.FileNotFoundException {
+        /*
+        Function prints out the commands for the user's convenience
+         */
+        Scanner input = new Scanner(new File("Instructions.txt"));
+        while (input.hasNextLine())
+        {
+           System.out.println(input.nextLine());
+        }
+    }
+
+    public static int login(String username, String password){
+        /*
+        Input: The username && password
+        Output: The username's userId or -1 if null and void
+        */
+
+        for(int i = 1; i < userDict.size()+1; i++) {
```

Comment:
reconsider a more efficient user login flow

Reply:
User Login Flow is now O(1) rather than O(n)